api/src/main/java/com/datatorrent/api/ControlTupleEnabledSink.java

```diff
@@ -18,7 +18,7 @@
  */
 package com.datatorrent.api;
 
-import org.apache.apex.api.UserDefinedControlTuple;
+import org.apache.apex.api.ControlTuple;
```

Comment:
I don't agree with the change. IMO, UserDefinedControlTuple name is more descriptive and properly reflects the intention of the interface. Please discuss at dev@apex.